style.css

```diff
@@ -0,0 +1,470 @@
+* {
+	box-sizing: border-box;
+}
+
+
+body {
+  margin: 0;
+  background: #aabbfb;
+  font-family: "Source Sans Pro";
+  font-size: 16px;
+}
+
+.nav-drop {
+	display: none;
+}
+
+.dropbtn {
+	background-color: #fbeaaa;
+	color:  black;
+	padding: 5.5px;
+	font-size: 1.8rem;
+	border: 0.5px solid;
+	padding-right: 20px;
+	padding-left: 20px;
+	padding-top: 6.5px;
+	margin-bottom: 5px;
+	margin-top: 1px;
+	border-radius: 8px;
+	margin-left: 5px;
+}
+
+.dropdown-content {
+	display: none;
+	position: absolute;
+	background-color: #aae4fb;
+	width: 108px;
+	box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+	z-index: 2;
+	margin-top: -5px;
+	margin-left: 5px;
+	border-radius: 8px;
+}
+
+.dropdown-content a {
+	color: black;
+	padding: 12px 16px;
+	text-decoration: none;
+	display: block;
+	border-bottom: 0.5px solid #000000;
+}
+
+.dropdown-content a:nth-child(4){
+	border: none;
+}
+
+.dropdown-content a:hover {
+	background-color:  #c2aafb
+}
+
+.nav-drop:hover .dropdown-content {
+	display: block
+}
+
+.nav-drop:hover .dropbtn {
+	background-color: #c2aafb;
+}
+
+.nav-container {
+	display: flex;
+	background-color: #fbeaaa;
+	justify-content: space-between;
+	height: 50px;
+	text-align: center;
+	width: 100%;
+}
+
+.nav-links {
+	display: flex;
+	justify-content: space-between;
+	text-align: center;
+	margin-left: 10px;
+}
+
+.nav-links a {
+	text-decoration: none;
+	color: black;
+	padding: 15px;
+	font-size: 18px;
+	font-family: Helvetica;
+	padding-top: 16px;
+	transition: transform .2s;	
+}
+
+.nav-links a:hover {
+	transform: scale(1.2);
+}
+
+.nav-cv {
+	display: flex;
+	margin-right: 10px;
+	width: 130px;
+	height: 40px;
+	margin-top: 5px;
+	justify-content: center;
+	align-items: center;
+	text-align: center;
+}
+
+.nav-cv a {
+	text-decoration: none;
+	color: black;
+	font-size: 1.1rem;
+	font-family: Helvetica;
+	height: 100%;
+	width: 100%;
+	border: 3px solid black;
+	border-radius: 8px;
+	padding: 5px;
+	padding-top: 8px;
+}
+
+.nav-cv a:hover {
+	color: #aabbfb;
+	border: 3px solid #aabbfb;
+}
+
+header {
+	display: flex;
+	width: 100%;
+	height: 800px;
+	background-image: url("coffee.jpg");
+	background-repeat: no-repeat;
+	background-size: cover;
+	background-position: center;
+	margin-top: 0px;
+	margin-bottom: 20px;
+	text-align: center;
+	align-items: center;
+	justify-content: center;
+}
+
+h1 {
+	font-family: "Josefin Sans";
+	font-size: 4rem;
+	color: #000000;
+	margin-top: 50px;
+}
+
+p {
+	font-family: "Montserrat";
+}
+
+h3 {
+	font-family: "Montserrat";
+	font-size: 2rem;
+}
+
+h5 {
+	font-family: "Montserrat";
+	font-size: 1.0rem;
+}
+
+.about-container {
+	display: flex;
+	justify-content: center;
+	flex-direction: row;
```

Comment:
These two properties default to center and row, so can be removed

Reply:
I have removed the flex-direction, but the removal of justify makes the section move to the left. Should I leave it in or fix another way?